### DISCUSSION TITLE
The indicator has an erroneous orientation in the Energy Impact.

### DISCUSSION THREAD

**Discussion by 17LingShan** [tauri-apps/tauri#8583] [General]
![image](https://github.com/tauri-apps/tauri/assets/93035991/43ff5d2d-f714-4edf-b6d2-7946090b6e9a)
In High average CPU usage, I found that the indicator in the Energy Impact had an erroneous orientation.
---little tail---
I'm not a native English speaker, so sometimes that may cause some concern. Please point out any errors so that I can improve my English writting skills.

**Comment by FabianLars** [comment]
Doesn't look like an error to me. Your app seems to be really JS heavy and measuring tools like these usually doesn't like that because they are meant to measure websites (typically with mobile in mind), not desktop apps.

In my opinion this is not _that_ much of an issue in desktop apps compared to websites but of course it can still be optimized (for example by moving some of the heavy lifting to the rust side instead).

Of course what i said doesn't really matter much if your app constantly has an insane cpu usage, that would of course be an issue you'd have to look into.
